README.md
in2master
=========

This extension provides you with configuration setup via TSConfig, TypoScript, HTML5 and CSS files for new TYPO3 4.X installations

Manual
======
[in2master Introduction](https://docs.google.com/document/d/1YGmBgy19CG37QtTzPw6skDqfJcArCrlHfJGXLUH9bEI/edit "in2master Introduction")

CHANGELOG.md
# CHANGELOG in2master

## master
* Remove excess README.txt

ext_tables.php
<?php
if (!defined('TYPO3_MODE')) {
	die ('Access denied.');
}

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/main', 'in2: 010 Config, General');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/performance', 'in2: 012 Config, Performance');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/language/german', 'in2: 020 Deutsch primäre Sprache');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/language/english', 'in2: 020 English primary Language');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/meta', 'in2: 030 SEO, Meta');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/headerComment', 'in2: 040 Header Comment');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/content', 'in2: 050 Content, Get*');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/logo', 'in2: 052 Content, Logo');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/nav/default', 'in2: 060 Menu, Default');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/nav/breadcrumb', 'in2: 061 Menu, Breadcrumb');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/nav/top', 'in2: 062 Menu, Top');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/nav/footer', 'in2: 063 Menu, Footer');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/nav/header', 'in2: 064 Menu, Header');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/nav/sidebar', 'in2: 065 Menu, Sidebar');

t3lib_extMgm::addStaticFile($_EXTKEY, 'Configuration/TypoScript/page', 'in2: 100 Page, Simple HTML5');

?>

ext_localconf.php
<?php
if ( ! defined('TYPO3_MODE') ) {
	die ('Access denied.');
}

// add settings from ext_conf_template.txt
$confArr	= unserialize( $GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['in2master'] );

if ( 1 == $confArr['enableMinimizedRTE'] ) {
	t3lib_extMgm::addPageTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:in2master/Configuration/TSConfig/rte.ts">');
}

if ( 1 == $confArr['enableDefaultPageTSconfig'] ) {
	t3lib_extMgm::addPageTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:in2master/Configuration/TSConfig/page.ts">');
}

if ( 1 == $confArr['enableDefaultUserTSconfig'] ) {
	t3lib_extMgm::addUserTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:in2master/Configuration/TSConfig/user.ts">');
}

// nice to have
$TYPO3_CONF_VARS['BE']['accessListRenderMode'] = 'checkbox';
$TYPO3_CONF_VARS['BE']['explicitADmode'] = 'explicitAllow';
$TYPO3_CONF_VARS['BE']['forceCharset'] = 'utf-8';
$TYPO3_CONF_VARS['BE']['maxFileSize'] = '100000';
$TYPO3_CONF_VARS['BE']['sessionTimeout'] = '36000';
$TYPO3_CONF_VARS['BE']['warning_email_addr'] = 'support@example.com';
$TYPO3_CONF_VARS['BE']['warning_mode'] = '2';
$TYPO3_CONF_VARS['FE']['forceCharset'] = 'utf-8';
$TYPO3_CONF_VARS['FE']['permalogin'] = '1';
$TYPO3_CONF_VARS['GFX']['enable_typo3temp_db_tracking'] = true;
$TYPO3_CONF_VARS['GFX']['thumbnails_png'] = '3';
$TYPO3_CONF_VARS['SYS']['lang']['cache']['clear_menu'] = true;
$TYPO3_CONF_VARS['SYS']['loginCopyrightShowVersion'] = false;
$TYPO3_CONF_VARS['SYS']['maxFileNameLength'] = '255';
$TYPO3_CONF_VARS['SYS']['textfile_ext'] = 'txt,pdf,html,htm,css,inc,php,php3,tmpl,js,sql';

// helpers
// uncomment to direct BE users to the frontend for editing
// $TYPO3_CONF_VARS['BE']['interfaces'] = 'backend,frontend';

// peformance
$TYPO3_CONF_VARS['BE']['compressionLevel'] = '5';
$TYPO3_CONF_VARS['BE']['versionNumberInFilename'] = true;
$TYPO3_CONF_VARS['FE']['compressionLevel'] = '5';
$TYPO3_CONF_VARS['FE']['debug'] = false;
$TYPO3_CONF_VARS['FE']['versionNumberInFilename'] = true;
$TYPO3_CONF_VARS['SYS']['belogErrorReporting'] = false;
$TYPO3_CONF_VARS['SYS']['enableDeprecationLog'] = '';
$TYPO3_CONF_VARS['SYS']['sqlDebug'] = false;
$TYPO3_CONF_VARS['SYS']['syslogErrorReporting'] = '0';
$TYPO3_CONF_VARS['SYS']['systemLogLevel'] = '4';

if ( 1 == $confArr['enableDeveloperMode'] ) {
	// TODO
	// create dev.ts
	// clear cache on every page load
	//
	// t3lib_extMgm::addPageTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:in2master/Configuration/TSConfig/dev.ts">');
	// Seems not possible as the following needs to be in localconf.php directly for processing
	// $TYPO3_CONF_VARS['EXT']['extList'] .= ',func,tstemplate_objbrowser,tstemplate_analyzer,func_wizards,lowlevel,beuser,info_pagetsconfig,reports,extension_builder';
	// $TYPO3_CONF_VARS['EXT']['extList_FE'] .= ',extension_builder';
	// $TYPO3_CONF_VARS['EXT']['extConf']['extension_builder'] = 'a:3:{s:15:"enableRoundtrip";s:1:"1";s:15:"backupExtension";s:1:"1";s:9:"backupDir";s:35:"uploads/tx_extensionbuilder/backups";}';
	$TYPO3_CONF_VARS['FE']['debug'] = true;
	$TYPO3_CONF_VARS['SYS']['belogErrorReporting'] = true;
	$TYPO3_CONF_VARS['SYS']['devIPmask'] = '192.168.*,127.0.0.1';
	$TYPO3_CONF_VARS['SYS']['displayErrors'] = '1';
	$TYPO3_CONF_VARS['SYS']['enableDeprecationLog'] = true;
	$TYPO3_CONF_VARS['SYS']['loginCopyrightShowVersion'] = true;
	$TYPO3_CONF_VARS['SYS']['sqlDebug'] = '1';	
	$TYPO3_CONF_VARS['SYS']['syslogErrorReporting'] = '1';
	$TYPO3_CONF_VARS['SYS']['systemLog'] = 'error_log,,1';
	$TYPO3_CONF_VARS['SYS']['systemLogLevel'] = '1';
}

if ( 1 == $confArr['enableServerSample'] ) {
	$TYPO3_CONF_VARS['GFX']['gdlib_png'] = '1';
	$TYPO3_CONF_VARS['GFX']['im_combine_filename'] = 'composite';
	$TYPO3_CONF_VARS['GFX']['im_imvMaskState'] = '1';
	$TYPO3_CONF_VARS['GFX']['im_v5effects'] = '1';
}

// $TYPO3_CONF_VARS['EXT']['extConf']['realurl'] = 'a:5:{s:10:"configFile";s:35:"typo3conf/ext/in2master/realurl.php";s:14:"enableAutoConf";s:1:"0";s:14:"autoConfFormat";s:1:"0";s:12:"enableDevLog";s:1:"0";s:19:"enableChashUrlDebug";s:1:"0";}';
$TYPO3_CONF_VARS['EXT']['extConf']['em'] = 'a:4:{s:14:"showOldModules";s:1:"0";s:14:"inlineToWindow";s:1:"0";s:19:"displayMyExtensions";s:1:"1";s:17:"selectedLanguages";s:0:"";}';

?>

Configuration/TSConfig/rte.ts
#------------------------------------
# RTE Configuration
#------------------------------------
RTE {
	default {
		#------------------------------------
		# which css file to use
		#------------------------------------
		contentCSS =

		#------------------------------------
		# possible CSS classes for links
		#------------------------------------
		classesAnchor = btn, btn-success, icon-thumbs-up
		classesAnchor.default {
			file =
			mail = 
			page =
			url =
		}

		#------------------------------------
		# custom formatblock (blockstyle update is required)
		#------------------------------------
		buttons {
			blockstyle {
				tags {
#					p.allowedClasses = anchorlistinghead
				}
			}

			formatblock {
#				addItems = anchorlistinghead

				items {
#					anchorlistinghead {
#						label = Anchorlisting Headline
#						addClass = anchorlistinghead
#						tagName = p
#					}
				}
			}
		}

		#------------------------------------
		# edit options for tables (cellspacing/ cellpadding / border)
		#------------------------------------
		disableAlignmentFieldsetInTableOperations = 1
		disableBordersFieldsetInTableOperations = 1
		disableColorFieldsetInTableOperations = 1
		disableLayoutFieldsetInTableOperations = 1
		disableSpacingFieldsetInTableOperations = 1

		#------------------------------------
		# This disables the RTE context menu so that spelling checking can be done with some browsers' context menu
		#------------------------------------
		disableContextMenu = 1

		#------------------------------------
		# Markup options
		#------------------------------------
		enableWordClean = 1
		removeComments = 1
		removeTags = font
		removeTagsAndContents = 
		removeTrailingBR = 1

		enablePersonalDicts = 1

		#------------------------------------
		# hide header levels and other stuff
		#------------------------------------
		hidePStyleItems =

		#------------------------------------
		# Keeps RTE Icons together in function groups
		#------------------------------------
		keepButtonGroupTogether = 1

		#------------------------------------
		# show statusbar
		#------------------------------------
		showStatusBar =  1

		#------------------------------------
		# Table decalarations in RTE Toolbar
		#------------------------------------
		hideTableOperationsInToolbar = 0
		keepToggleBordersInToolbar = 1

		#------------------------------------
		# Shows all CSS classes in contentCSS file
		#------------------------------------
		showTagFreeClasses = 1

		#------------------------------------
		# Do not allow insertion of the following tags
		#------------------------------------
		hideTags < RTE.default.removeTags

		#------------------------------------
		# Show / hide buttons
		# cut / copy / paste are not shown in Chrome and Safari
		#
		# options
		# copy, cut, paste, pastetoggle, pastebehaviour, pasteastext,
		# textstyle, formatblock,  textstylelabel, blockstyle, blockstylelabel, 
		# bold, left, center, right, orderedlist, unorderedlist, insertcharacter,
		# link, removeformat, table, toggleborders, tableproperties, 
		# rowproperties, rowinsertabove, rowinsertunder, rowdelete, rowsplit,
		# columninsertbefore, columninsertafter, columndelete, columnsplit,
		# cellproperties, cellinsertbefore, cellinsertafter, celldelete, cellsplit, 
		# cellmerge, findreplace, insertcharacter, undo, redo
		# fontstyle, fontsize, strikethough, line, lefttoright, righttoleft, 
		# textcolor, bgcolor, textindicator, emoticon, user, spellcheck, 
		# chMode, inserttag, outdent, indent, justifyfull, subscript, superscript,
		# acronym, italic, underline, image, showhelp, about
		#------------------------------------
		showButtons := addToList(orderedlist, unorderedlist, insertcharacter, link, image, removeformat, chMode, spellcheck)
		showButtons := removeFromList(about)

		hideButtons =

		#------------------------------------
		# Add styles Left, center and right alignment of text in paragraphs (and cells).
		# it is possible to add td.align* to this list
		#------------------------------------
		inlineStyle.text-alignment (
			p.align-left, h1.align-left, h2.align-left, h3.align-left, h4.align-left, h5.align-left, h6.align-left { text-align: left; }
			p.align-center, h1.align-center, h2.align-center, h3.align-center, h4.align-center, h5.align-center, h6.align-center { text-align: center; }
			p.align-right, h1.align-right, h2.align-right, h3.align-right, h4.align-right, h5.align-right, h6.align-right { text-align: right; }
		)

		#------------------------------------
		# Use stylesheet file rather than the above mainStyleOverride and inlineStyle properties to style the contents
		#------------------------------------
		ignoreMainStyleOverride = 1

		proc {
			#------------------------------------
			# list of allowed tags
			#
			# options
			# table, tbody, thead, tr, th, td, h1, h2, h3, h4, h5, h6, div, p, 
			# br, span, ul, ol, li, re, blockquote, strong, em, b, i, u, sub, 
			# sup, strike, a, img, nobr, hr, tt, q, cite, abbr, acronym, center
			#
			# Allow embed (Youtube) HTML tags in the RTE
			#------------------------------------
			allowTags := addToList(object,param,embed,iframe,script)

			#------------------------------------
			# tags allowed outside from p and div
			#------------------------------------
			allowTagsOutside := addToList(object,embed,iframe,script)

			#------------------------------------
			# list of denied tags
			#------------------------------------
			denyTags < RTE.default.removeTags

			#------------------------------------
			# as the name says ;-)
			#------------------------------------
			dontConvBRtoParagraph = 1

			#------------------------------------
			# allowed attributes in p and div tags
			#------------------------------------
			keepPDIVattribs = align,class,style,id

			#------------------------------------
			# List all of your class selectors that are allowed on the way to the database
			# custom CSS classes for formatblock
			# allowedClasses := addToList(anchorlistinghead)
			#------------------------------------
			allowedClasses =

			#------------------------------------
			# HTML parser settings
			#------------------------------------
			HTMLparser_rte {
				#------------------------------------
				# allowed / denied tags (should be usually the same as in proc.*, otherwise it could be messed up)
				#------------------------------------
				allowTags < RTE.default.proc.allowTags
				denyTags < RTE.default.proc.denyTags
				removeTags < RTE.default.removeTags

				#------------------------------------
				# remove HTML comments
				#------------------------------------
				removeComments = 1

				#------------------------------------
				# Unmatched tags are removed (protect / 1 / 0)
				#------------------------------------
				keepNonMatchedTags = 0
			}

			#------------------------------------
			# modifications of the content on while saving to database
			#------------------------------------
			entryHTMLparser_db = 1
			entryHTMLparser_db {
				#------------------------------------
				# allowed / denied tags (should be usually the same as in proc.*, otherwise it could be messed up)
				#------------------------------------
				allowTags < RTE.default.proc.allowTags
				denyTags < RTE.default.proc.denyTags
				removeTags < RTE.default.removeTags

				#------------------------------------
				# remove all attributes from following tags
				#------------------------------------
				noAttrib = b, i, u, strike, sub, sup, strong, em, quote, blockquote, cite, tt, br, center

				#------------------------------------
				# remove tags if there is no attribute available
				#------------------------------------
				rmTagIfNoAttrib = span,div

				#------------------------------------
				# Encode HTML's special characters
				#------------------------------------
				# htmlSpecialChars = 1

				#------------------------------------
				# how to handle tags and attributes
				#------------------------------------
				tags {
					#------------------------------------
					# allow align attribute
					#------------------------------------
					p {
						fixAttrib.align.unset = 1
						allowedAttribs := addToList(class,style,align)
					}

					div.fixAttrib.align.unset >

					hr.allowedAttribs = class

					#------------------------------------
					# replace b and i tags
					#------------------------------------
					b.remap = strong
					i.remap = em

					#------------------------------------
					# allow img tags
					#------------------------------------
					img >
				}
			}
		}

		#------------------------------------
		# All classes have to be in RTE.default.proc.allowedClasses too.
		#------------------------------------
		# possible CSS classes for paragraphs
		#------------------------------------
		classesParagraph = align-left, align-center, align-right

		#------------------------------------
		# possible CSS classes for spans
		#------------------------------------
		classesCharacter =

		#------------------------------------
		# possible CSS classes for images
		#------------------------------------
		classesImage =

		#------------------------------------
		# possible CSS classes for tables
		#------------------------------------
		classesTable =
		classesTR =
		classesTH =
		classesTD =
	}
}

#------------------------------------
# Use same processing as on entry to database to clean content pasted into the editor
#------------------------------------
RTE.default.enableWordClean.HTMLparser < RTE.default.proc.entryHTMLparser_db

#------------------------------------
# FE RTE configuration (htmlArea RTE only)
#------------------------------------
RTE.default.FE < RTE.default
RTE.default.FE.userElements >
RTE.default.FE.userLinks >


Configuration/TSConfig/page.ts
# http://typo3.org/documentation/document-library/references/doc_core_tsconfig/4.3.2/view/1/5/#id2506208
TCEFORM {
	pages {
		#------------------------------------------------
		# page-uid, where backend layouts are saved
		#------------------------------------------------
		backend_layout.PAGE_TSCONFIG_ID=2
		backend_layout_next_level.PAGE_TSCONFIG_ID=2

		#-------------------------------------------------
		# disable fields of pages, that are not configured
		# 0 for off, 1 for on
		#-------------------------------------------------
		abstract.disabled = 1
		alias.disabled = 1
		author.disabled = 0
		cache_timeout.disabled = 1
		content_from_pid.disabled = 1
		description.disabled = 0
		email.disabled = 0
		fe_login_mode.disabled = 1
		# image.config.disable_controls = upload
		is_siteroot.disabled = 0
		keywords.disabled = 0
		l18n_cfg.disabled = 1
		lastUpdated.disabled = 1
		layout.disabled = 1
		module.disabled = 1
		newUntil.disabled = 1
		no_cache.disabled = 0
		php_tree_stop.disabled = 1
		subtitle.disabled = 1
		target.disabled = 1
		tx_damtree_dam_cats.disabled = 1
	}

	tt_content {
		header_layout {
			altLabels.1 = 1st Level
			altLabels.2 = 2nd Level
			altLabels.3 = 3rd Level
			altLabels.4 = 4th Level
			altLabels.5 = 5th Level
		}

		# example for adding and remove header_layout items
		# header_layout.removeItems = 100
		# header_layout.addItems.6 = 6th Level Header

		#-------------------------------------------------
		# disable fields of tt_content, that are not configured
		# 0 for off, 1 for on
		#-------------------------------------------------
		colPos.disabled = 0
		date.disabled = 1
		header_position.disabled = 1
		image_compression.disabled = 1
		image_effects.disabled = 1
		layout.disabled = 1
		linkToTop.disabled = 1
		sectionIndex.disabled = 1
		section_frame.disabled = 0
		spaceAfter.disabled = 1
		spaceBefore.disabled = 1
		subheader.disabled = 1
		sys_language_uid.disabled = 0
		tx_gridelements_columns.disabled = 0
		tx_gridelements_container.disabled = 0

		#------------------------------------
		# RTE width in fullscreen view
		#------------------------------------
		bodytext.RTEfullScreenWidth = 80%
	}

	tt_news {
		#-------------------------------------------------
		# disable fields of tt_news, that are not configured
		# 0 for off, 1 for on
		#-------------------------------------------------
		author.disabled = 0
		editlock.disabled = 1
		fe_group.disabled = 0
		image.disabled = 0
		imagealttext.disabled = 0
		imagecaption.disabled = 0
		imagetitletext.disabled = 0
		keywords.disabled = 0
		links.disabled = 0
		news_files.disabled = 0
		related.disabled = 0
		sys_language_uid.disabled = 0
		# type.removeItems = 2

		bodytext.RTEfullScreenWidth < TCEFORM.tt_content.bodytext.RTEfullScreenWidth
		# image.config.disable_controls = upload
	}
}

# http://typo3.org/documentation/document-library/references/doc_core_tsconfig/4.3.2/view/1/5/#id2507271
TSFE.jumpUrl_transferSession = 1

TCEMAIN {
	translateToMessage = Please translate `%s`

	# Don't hide newly copied pages or content. Prevent `copy X` on copy
	default {
		disablePrependAtCopy = 1
		disableHideAtCopy = 1
	}

	table {
		pages < TCEMAIN.default
		tt_content < TCEMAIN.default
	}

	# default be usergroup when creating new pages (every beuser should be in this group!)
	permissions {
		groupid = 1
		user = show,edit,delete,new,editcontent
		group = show,edit,delete,new,editcontent
		everybody =
	}
}

# Set the default label and flag
mod {
	SHARED {
		# Show only the chosen columns in the page module
		# 0=normal, 1=left, 2=right, 3=border
		# colPos_list = 1,0,2
		colPos_list = 0,2

		# defaultLanguageLabel = English
		# defaultLanguageFlag = gb.gif

		# defaultLanguageLabel = Deutsche
		# defaultLanguageFlag = de.gif
	}

	web_list {
		# edit record just by clicking on title in list-view
		clickTitleMode = edit

		# alternating colors for records in list-view
		alternateBgColors = 1

		itemsLimitPerTable = 20
		itemsLimitSingleTable = 100
	}

	xMOD_alt_doc {
		disableDocSelector = 1
		disableCacheSelector = 1
	}

	web_layout {
		defLangBinding = 1
	}
}

# match kept columns
mod.web_layout.defLangBinding = 1


Configuration/TSConfig/user.ts
options {
	clearCache {
		pages = 1
		all = 1
	}

	saveDocNew = 1

	# don't clear clipboard
	saveClipboard = 1

	# Reduces the amount of clipboards from 4 to X:
	clipboardNumberPads = 2

	shortcutFrame = 1
	
	# Enable upload field
	uploadFieldsInTopOfEB = 1

	# Enable folder create
	createFoldersInEB = 1

	## Set the default spelling ability of the check speller for all users
	## options: ultra, fast, normal, bad-spellers
	HTMLAreaPspellMode = normal

	## Enable the personal dictionary feature of the check speller by default for all users
	enablePersonalDicts = 1

	# show pageID in page tree title
	pageTree.showPageIdWithTitle = 0

	# show navigation title in page tree if any
	pageTree.showNavTitle = 1

	# options.contextMenu.pageTree.disableItems = history,edit_access,move_wizard,new_wizard,db_list,tx_impexp_clickmenu,info,perms, versioning

	options.RTEkeyList = *
	# options.RTEkeyList = formatblock, bold, italic, orderedlist, unorderedlist, outdent, indent, link, unlink, blockstylelabel, blockstyle, textstylelabel, textstyle
}

mod {
	web_list {
		# Hide static_info_tables records on the rootpage
		hideTables = static_template,static_countries,static_country_zones,static_currencies,static_languages,static_territories,static_taxes,static_markets
	}

	wizards {
		# http://typo3.org/documentation/document-library/core-documentation/doc_core_tsconfig/4.3.2/view/1/5/#id2505051
		# http://buzz.typo3.org/people/steffen-kamper/article/the-new-content-element-wizard-in-version-43/
		newContentElement {
			renderMode = tabs

			wizardItems {
				common {
					elements {
						header {
							icon = gfx/c_wiz/regular_header.gif
							title = Header
							description = A simple header.
							tt_content_defValues {
								CType = header
							}
						}
					}

					# original: text,textpic,image,bullets,table
					show := addToList(header)

					#--------------------------------------
					# show only those elements, rest is possible via RTE
					#--------------------------------------
					show := removeFromList(text,image,bullets,table)
				}

				plugins {
					show =
				}

				special {
					# original: uploads,media,menu,html,div
					show := removeFromList(uploads,div)
				}
			}
		}
	}
}

ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext "in2master".
#
# Auto generated 27-06-2012 11:38
#
# Manual updates:
# Only the data in the array - everything else is removed by next
# writing. "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'in2code Master',
	'description' => 'This extension provides you with configuration setup via TSConfig, TypoScript, HTML5 and CSS files for new TYPO3 4.X installations.',
	'category' => 'be',
	'author' => 'Michael Cannon',
	'author_email' => 'info@in2code.de',
	'author_company' => 'in2code GmbH',
	'shy' => '',
	'priority' => '',
	'module' => '',
	'state' => 'alpha',
	'internal' => '',
	'uploadfolder' => 0,
	'createDirs' => '',
	'modify_tables' => '',
	'clearCacheOnLoad' => 0,
	'lockType' => '',
	'version' => '0.0.0',
	'constraints' => array(
		'depends' => array(
			'cms' => '',
			'extbase' => '',
			'fluid' => '',
		),
		'conflicts' => array(
		),
		'suggests' => array(
			'js_css_optimizer' => '',
			'nc_staticfilecache' => '',
			'realurl' => '',
			'tq_seo' => '',
			'wt_spamshield' => '',
		),
	),
	'_md5_values_when_last_written' => 'a:47:{s:21:"ExtensionBuilder.json";s:4:"e7f8";s:10:"README.txt";s:4:"a34b";s:10:"_.htaccess";s:4:"35b9";s:21:"ext_conf_template.txt";s:4:"e882";s:12:"ext_icon.gif";s:4:"adb8";s:17:"ext_localconf.php";s:4:"ee53";s:14:"ext_tables.php";s:4:"5fdb";s:14:"ext_tables.sql";s:4:"d41d";s:11:"realurl.php";s:4:"ce32";s:44:"Configuration/ExtensionBuilder/settings.yaml";s:4:"e145";s:30:"Configuration/TSConfig/page.ts";s:4:"6906";s:29:"Configuration/TSConfig/rte.ts";s:4:"86d7";s:30:"Configuration/TSConfig/user.ts";s:4:"315f";s:49:"Configuration/TypoScript/contentGet/constants.txt";s:4:"62c9";s:45:"Configuration/TypoScript/contentGet/setup.txt";s:4:"aed9";s:48:"Configuration/TypoScript/headerComment/setup.txt";s:4:"50ff";s:51:"Configuration/TypoScript/language/english/setup.txt";s:4:"4c05";s:50:"Configuration/TypoScript/language/german/setup.txt";s:4:"51b9";s:43:"Configuration/TypoScript/logo/constants.txt";s:4:"b673";s:39:"Configuration/TypoScript/logo/setup.txt";s:4:"6c79";s:43:"Configuration/TypoScript/main/constants.txt";s:4:"bed0";s:39:"Configuration/TypoScript/main/setup.txt";s:4:"584d";s:51:"Configuration/TypoScript/menus/breadcrumb/setup.txt";s:4:"84b6";s:48:"Configuration/TypoScript/menus/default/setup.txt";s:4:"d155";s:51:"Configuration/TypoScript/menus/footer/constants.txt";s:4:"eb6f";s:51:"Configuration/TypoScript/menus/footer/lib.footer.ts";s:4:"80e8";s:47:"Configuration/TypoScript/menus/footer/setup.txt";s:4:"82d9";s:47:"Configuration/TypoScript/menus/header/setup.txt";s:4:"9c7c";s:48:"Configuration/TypoScript/menus/sidebar/setup.txt";s:4:"3a41";s:48:"Configuration/TypoScript/menus/top/constants.txt";s:4:"25e1";s:44:"Configuration/TypoScript/menus/top/setup.txt";s:4:"4b63";s:39:"Configuration/TypoScript/meta/setup.txt";s:4:"0661";s:43:"Configuration/TypoScript/page/constants.txt";s:4:"f11e";s:39:"Configuration/TypoScript/page/setup.txt";s:4:"fbea";s:50:"Configuration/TypoScript/performance/constants.txt";s:4:"c6f7";s:46:"Configuration/TypoScript/performance/setup.txt";s:4:"1ae1";s:40:"Resources/Private/Language/locallang.xml";s:4:"c7f8";s:43:"Resources/Private/Language/locallang_db.xml";s:4:"0c0e";s:48:"Resources/Private/Layouts/in2master_default.html";s:4:"fffd";s:42:"Resources/Private/html/in2master_main.html";s:4:"7e4d";s:35:"Resources/Public/Icons/relation.gif";s:4:"e615";s:39:"Resources/Public/css/in2master_main.css";s:4:"8c6c";s:30:"Resources/Public/css/print.css";s:4:"c079";s:44:"Resources/Public/css/tx_cssstyledcontent.css";s:4:"5d37";s:40:"Resources/Public/images/in2code_logo.png";s:4:"ac9c";s:14:"doc/manual.pdf";s:4:"2ec2";s:14:"doc/manual.sxw";s:4:"20fb";}',
);

?>